src/visualBuilder/utils/constants.ts

```diff
@@ -16,6 +16,8 @@ export const TOOLBAR_EDGE_BUFFER = 8;
 
 export const DATA_CSLP_ATTR_SELECTOR = "data-cslp";
 
+export const WAIT_FOR_NEW_INSTANCE_TIMEOUT = 4000;
```

Comment:
I think we need a better name. This is the the timeout to wait for any change happening in the canvas right? Probably `WAIT_FOR_VARIANT_CHANGES_TIMEOUT`

Reply:
ok